source/includes/v2/_odsql.md

```diff
@@ -804,3 +804,34 @@ The count of pattern letters determine the format.
 ##### Format:
 `group_by=date_format(<string_literal>)`
 in which `<string_literal>` contains a date format
+
+## Order by clause
+
+> Order by examples
+```python
```

Comment:
It doesn't look like Python though!

Reply:
But joke aside, looking at it again, it provides better highlighting than nothing and could be added for the other ODSQL snippets in this file (currently there are written as ``` plain-text )

That or  ``` mysql that also kinda works

Reply:
Indeed! I tried SQL also and I like it even more. I will change it to this, and even add another PR to change it everywhere.